### ISSUE TITLE
Structure for allowing new initiatives

### ISSUE THREAD

**Issue by d-ataman** [sigturk/sigturk#81]
How can we use a better communication medium for new research ideas to be organized and accessing ongoing projects or potential collaborators are easier? Github can be very structured but maybe we can prepare some instructions for new members to adopt using it? We can discuss in the next monthly.

**Comment by d-ataman** [comment]
I think the best would be to use our own webpage to keep stats on each project but would need still some level of weekly maintenance (linking after meeting notes, etc.) We should already have this capacity in our infrastructure so maybe it's feasible to automate the creation, and maintenance of new working groups/projects?

**Comment by mehmetoguzderin** [comment]
I totally agree, we can bring more automation based on process definition. It'd increase appeal of collaborating with SIGTURK a lot

**Comment by d-ataman** [comment]
in the general meeting we decided to make a short public description on the webpage, and include instructions and directions to follow up on their project ideas and propose papers for the reading group.

**Comment by mehmetoguzderin** [comment]
@d-ataman Is there a draft that I can bring to website? Or is it to be written yet?
